sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -464,8 +464,8 @@ def _op_to_template(op: BaseOp, pipelinerun_output_artifacts={}, enable_artifact
                 output_annotation = pipelinerun_output_artifacts.get(processed_op.name, [])
                 output_annotation.append(
                     {
-                        'name': output_artifact['name'],
-                        'path': output_artifact['path'],
+                        'name': output_artifact.get('name'),
```

Comment:
can `output_artifact['name']` or `output_artifact['path']` ever be missing here (raising a `KeyError`)? If so, should the `get` return a default value like empty string `""`?

`output_artifact['name']` is used 2 lines down with direct key access as opposed to using `dict.get()`

Reply:
It was a merge from feng's PR yesterday, I can replace all of them to use .get() with defaults to make it consistent.